libs/libideviceactivation/Makefile

```
@@ -0,0 +1,99 @@
+#
+# Copyright (C) 2012-2024 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=libideviceactivation
+PKG_VERSION:=1.1.2
+PKG_RELEASE:=1
+
+PKG_SOURCE_URL:=https://github.com/libimobiledevice/$(PKG_NAME).git
+PKG_SOURCE_PROTO:=git
```

```suggestion
PKG_SOURCE_PROTO:=git
PKG_SOURCE_URL:=https://github.com/libimobiledevice/libideviceactivation.git
```
please leave repo url as-is for better readability
the same applies below

---

I applied this to all packages: PROTO before URL, URL as is.